src/HH4b/xsecs.py

```diff
@@ -96,15 +98,44 @@
 xsecs["ggZH_Hto2B_Zto2Q_M-125"] = 0.1360 * BR_ZQQ * BR_HBB
 xsecs["ttHto2B_M-125"] = 0.5700 * BR_HBB
 
+# Triple-higgs
+# ggHHHto6B_13TeV         0.0894e-3 (not including BR_HBB)
+xsecs["HHHTo6B_c3_0_d4_0"] = 3.707e-05 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_0_d4_99"] = 0.005855 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_0_d4_minus1"] = 4.117e-05 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_19_d4_19"] = 0.1476 * BR_HBB * BR_HBB * BR_HBB
+# SM sample
+xsecs["HHHTo6B_c3_1_d4_0"] = 2.908e-05 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_1_d4_2"] = 1.616e-05 * BR_HBB * BR_HBB * BR_HBB
+# xsecs["HHHTo6B_c3_2_d4_minus1"] =
+# xsecs["HHHTo6B_c3_4_d4_9"] =
+xsecs["HHHTo6B_c3_minus1_d4_0"] = 0.0001127 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_minus1_d4_minus1"] = 0.0001087 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_minus1p5_d4_minus0p5"] = 0.0001941 * BR_HBB * BR_HBB * BR_HBB
+
 # Di-Higgs
-# TO- XCHECK
-xsecs["GluGlutoHHto4B_kl-1p00_kt-1p00_c2-0p00"] = 34.43e-3 * BR_HBB * BR_HBB  # 0.02964 in xsecdb
-xsecs["GluGlutoHHto4B_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_TSG"] = 34.43e-3 * BR_HBB * BR_HBB
-xsecs["GluGlutoHHto4B_kl-0p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_TSG"] = (
-    0.06648 * BR_HBB * BR_HBB
-)  # from xsecdb
-xsecs["GluGlutoHHto4B_kl-2p45_kt-1p00_c2-0p00_TuneCP5_13p6TeV_TSG"] = 0.01252 * BR_HBB * BR_HBB
-xsecs["GluGlutoHHto4B_kl-5p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_TSG"] = 0.08664 * BR_HBB * BR_HBB
+# TO- XCHECK (from xsecdb)
+hh = {
+    "GluGlutoHHto4B_kl-0p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV": 0.06648 * BR_HBB * BR_HBB,
+    "GluGlutoHHto4B_kl-0p00_kt-1p00_c2-1p00_TuneCP5_13p6TeV": 0.1492 * BR_HBB * BR_HBB,
+    "GluGlutoHHto4B_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV": 0.02964 * BR_HBB * BR_HBB,
```

Comment:
shouldn't this be 34.43e-3 * BR_HBB * BR_HBB?

Reply:
where did we get that number? I got 0.02964 from xsdb

Reply:
This says 34.43 https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWGHH#Latest_recommendations_for_gluon

Reply:
I don't see any for other kLs, but I guess that's not important right now.

Reply:
ah okay, go ahead and change it then